test/unit/matrix/test_matrix_view.cpp

```diff
@@ -38,6 +38,7 @@ template <typename Type>
 class MatrixViewTest : public ::testing::Test {
 public:
   const std::vector<CommunicatorGrid>& commGrids() {
+    EXPECT_FALSE(comm_grids.empty());
     return comm_grids;
   }
 };
```

Comment:
Minor suggestion: we may decide to create our test type to inherit from.

Something like

```cpp
struct TestWithCommGrids : public ::testing::Test {
  const std::vector<CommunicatorGrid>& commGrids() {
    EXPECT_FALSE(comm_grids.empty());
    return comm_grids;
  }
};
```

and then just have
```cpp
using MatrixViewTest = TestWithCommGrids; // or inherit from it
```

Reply:
Please check the latest commit.